lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -26,6 +26,7 @@
 #include "circt/Dialect/HW/HWVisitors.h"
 #include "circt/Dialect/SV/SVOps.h"
 #include "circt/Dialect/SV/SVVisitors.h"
+#include "circt/Dialect/Seq/SeqDialect.h"
```

Comment:
Is this from another branch? I don't think ExportVerilog needs to know about Seq.

Reply:
Thanks for spotting this!